src/xcode/ENA/ENA/Source/Services/HealthCertificate/HealthCertificateService.swift

```diff
@@ -465,7 +475,7 @@ class HealthCertificateService {
 
 						let signatureVerificationResult = self.signatureVerifying.verify(
 							certificate: healthCertificate.base45,
-							with: [],
+							with: self.dscListProvider.signingCertificates.value,
```

Comment:
Same here. The signingCertificates are fetched async, so at this point, it could happen that the signingCertificates are the default ones?

Reply:
Same as above :)